scripts/coder-dev.sh

```diff
@@ -55,4 +56,11 @@ coder)
 	;;
 esac
 
-exec "${CODER_DEV_BIN}" --global-config "${CODER_DEV_DIR}" "$@"
+runcmd="${CODER_DEV_BIN}"
+if [[ "${debug}" == 1 ]]; then
+	set -x
+	runcmd="dlv debug --headless --continue --listen 127.0.0.1:12345 --accept-multiclient ./cmd/coder --"
+fi
+
+# shellcheck disable=SC2086 # This is probably one of the few times you actually want this.
```

Comment:
Feel free to change or not change, just pointing out the option:

Alternatively, we could make `runcmd` and array (`runcmd=(dlv debug ...)`) and use `"${runcmd[@]}"` (quotes are OK here due to @, it essentially produces `"$1" "$2"` vs `$1 $2`).

Reply:
Added!